R/Auth.R

```diff
@@ -0,0 +1,75 @@
+#' Create a connection to an instance of WebAPI
+#' 
+#' This function creates a WebAPI connection object that can be used by other
+#' functions in this package to access WebAPI. This function will test 
+#' the connection as well as trigger authentication and store the auth
+#' session token.
+#'
+#' @template baseUrl 
+#' @param authMethod The method used for authentication to WebAPI. Options are
+#' \itemize{
+#'  \item{"db"}{Database authentication using Atlas/WebAPI built in auth}
+#'  \item{"AD"}{Active Directory}
+#'  \item{"windows"}{Windows}
+#'  \item{"none"}{No authentication. WebAPI security must be disabled.}
+#' }
+#' By default the correct authentication method will be guessed.
+#' @param keyringServiceName (Optional) Keyring stores passwords securely. 
+#'                           For some authentication methods it may be convenient 
+#'                           to pass keyring service name instead of entering a password.
+#'
+#' @return A WebApi connection object
+#' @export
+connectWebApi <- function(baseUrl, 
+                          authMethod = guessAuthMethod(),
+                          atlasUsername = NULL,
```

Comment:
We decided to avoid 'atlas' where possible, because this package is designed for webApi. The way we saw it in the design discussion was Atlas is a UI for WebApi. 

Recommend we call it webApiUserName

Reply:
See https://github.com/OHDSI/ROhdsiWebApi/commit/eb7e3ad80b4fe536861a99b104a781aa2d4bbd8a#diff-8312ad0561ef661716b48d09478362f3R12